modules/javafx.graphics/src/main/native-glass/mac/GlassApplication.m

```diff
@@ -753,6 +754,11 @@ - (jobject)getPlatformPreferences
     return platformSupport != nil ? [platformSupport collectPreferences] : nil;
 }
 
++ (BOOL)canStartNestedEventLoop
+{
+    return nestedRunLoopRunCount <= 250;
```

Comment:
How sure are you that 250 is a safe limit? Are there situations where we might not be able to do this many?

I recommend added a comment as to where the magic number "250" comes from. And maybe put it in a static const variable?

Reply:
I added a comment explaining where the number comes from. Since it's only used in this one spot putting it in a static const variable seemed redundant.

I'm not sure that 250 is a safe limit. I know that the JUnit test crashes if this value is set to 252 but succeeds at 251. I don't want to risk disturbing existing apps but since each count represents a runnable waiting for an event loop to exit even 200 seems like a high number. It might be prudent to set this to, say, 246 to give us a bit more headroom.

Reply:
I agree that it might be wise to set it a bit lower, so either 246 or even 240 seems reasonable.

Reply:
Set to 244 so I didn't have to alter the system test.